instance/models/openedx_instance.py

```diff
@@ -111,10 +115,7 @@ def domain(self):
 
         Returns external domain if present, otherwise falls back to internal domain.
         """
-        if self.external_lms_domain:
-            return self.external_lms_domain
-        else:
-            return self.internal_lms_domain
+        return self.get_domain('lms')
```

Comment:
@haikuginger I meant to ping Eugeny, not you.  I'm asking what the additional variables would buy us.  I think the problem here is that the structure of the data is encoded in the field names (see my other comment), but this would not be addressed by Eugeny's suggestion.